tests/controller_tests/flow_tests/flow_test_1.py

```diff
@@ -0,0 +1,34 @@
+import json
+from requests import post, delete
+
+
+add_card_json = {
+    "content": "hello world",
+    "type": "answer",
+}
+
+remove_card_json = {
+
+}
+
+get_cards_json = {
+    "type": "answer",
+}
+
+
+add_response = post(url="http://127.0.0.1:5000/add_card", json=add_card_json)
```

Comment:
Do I have this db? 
What do I need to do for it to work on my machine (didn't try just asking )?

Reply:
it should work on your machine after you just pip install sqlite3, call me if it does not work :)